packages/wdio-appium-service/src/launcher.ts

```diff
@@ -17,7 +17,7 @@ const DEFAULT_CONNECTION = {
 }
 
 export default class AppiumLauncher {
-    public logPath: string
+    public logPath?: string
```

Comment:
I think we can make all these parameters private and prefix them with an underscope
```suggestion
    private _logPath?: string
```

Reply:
fixed